resources/dns/records.tf

```diff
@@ -0,0 +1,379 @@
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_dmarc" {
+  name    = "_dmarc"
+  comment = "DMARC for communications"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DMARC1; p=reject; pct=100; rua=mailto:g5oqy2di@ag.dmarcian.com, mailto:admin@subspace.network; ruf=mailto:admin@subspace.network"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_domain_key" {
+  name    = "_domainkey"
+  comment = "DomainKeys authentication for mail"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DKIM1; o=~"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_google_domain_key" {
+  name    = "google._domainkey"
+  comment = "DomainKeys authentication for mail specific to Google mailserver"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=DKIM1; k=rsa; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAyqbz3f5O6pOl9LK3zOmKkczcK9aCdB34tSzDkA ZRD0g+OtBhlyWyzNeQGsrVianIfM9xyfZ7MVJK7sB/VGKIFIe5glb/Lh9tf/kLCRbkOnaafiXP5tOk4DC+mBpHOjyT0GgX5x4hxg oLmeJOrRLSu1niPQY/VqMtNsYa9+gAIo0YZnZeNi2w3FjWaslm5F7uI6mISdH3HZchqhzx2E6Ct+VJhLM0Ir+6v3qV5ylArtJzc+ PoTGdk65n5oq+Ioj9oTK4VmJunZ6jaU2Vimo+2TTAhBtRWQCa8Xy15/3kVZXiKJ4ddtPDnzSEval4er4SksIhtVYaxkeanU7pGbK w1SQIDAQAB"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mailserver_spf" {
+  name    = "subspace.network"
+  comment = "SPF record"
+  proxied = false
+  ttl     = 3600
+  type    = "TXT"
+  value   = "v=spf1 +a +a:ns1.us145.siteground.us include:_spf.google.com +mx ~all"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "mail" {
+  name    = "mail"
+  proxied = true
+  ttl     = 1
+  type    = "CNAME"
+  value   = "subspace.network"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+// subspace blog
+resource "cloudflare_record" "blog_0" {
+  comment = "Medium .blog Redirect #2"
+  name    = "blog"
+  proxied = false
+  ttl     = 3600
+  type    = "A"
+  value   = "162.159.152.4"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+resource "cloudflare_record" "blog_1" {
+  comment = "Medium .blog Redirect #1"
+  name    = "blog"
+  proxied = false
+  ttl     = 3600
+  type    = "A"
+  value   = "162.159.153.4"
+  zone_id = data.cloudflare_zone.cloudflare_zone.id
+}
+
+// subspace docs
+resource "cloudflare_record" "docs" {
+  name    = "docs"
+  comment = "Redirected docs to Github"
+  proxied = true
+  ttl     = 1
```

Comment:
TTL here and in some records below is still 1 second. Most DNS servers will ignore it anyway, why not specifying something more reasonable?

Reply:
for proxied records, cloudflare enforce to be 1